src/pages/settings/InitialSettingsPage.js

```diff
@@ -272,114 +258,117 @@ class InitialSettingsPage extends React.Component {
                 iconStyles={item.iconStyles}
                 shouldShowRightIcon
                 iconRight={item.iconRight}
-                badgeText={this.getWalletBalance(isPaymentItem)}
+                badgeText={getWalletBalance(isPaymentItem)}
                 fallbackIcon={item.fallbackIcon}
                 brickRoadIndicator={item.brickRoadIndicator}
                 floatRightAvatars={item.floatRightAvatars}
                 shouldStackHorizontally={item.shouldStackHorizontally}
                 floatRightAvatarSize={item.avatarSize}
-                ref={this.popoverAnchor}
+                ref={popoverAnchor}
                 shouldBlockSelection={Boolean(item.link)}
-                onSecondaryInteraction={!_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONTEXT_MENU_TYPES.LINK, e, item.link, this.popoverAnchor.current) : undefined}
+                onSecondaryInteraction={!_.isEmpty(item.link) ? (e) => ReportActionContextMenu.showContextMenu(CONTEXT_MENU_TYPES.LINK, e, item.link, popoverAnchor.current) : undefined}
             />
         );
-    }
+    };
 
-    toggleSignoutConfirmModal(value) {
-        this.setState({shouldShowSignoutConfirmModal: value});
-    }
+    const toggleSignoutConfirmModal = (value) => {
+        setShouldShowSignoutConfirmModal(value);
+    };
 
-    signOut(shouldForceSignout = false) {
-        if (!this.props.network.isOffline || shouldForceSignout) {
+    const signOut = (shouldForceSignout = false) => {
```

Comment:
We earlier had a binding which ignored the case for this method. Can you please fix either the method name or the method call?